pkg/util/httputil/json.go

```diff
@@ -77,6 +77,13 @@ type JSONResponseWriter struct {
 	Logger JSONResponseWriterLogger
 }
 
+func (w *JSONResponseWriter) WriteResponseWithCookies(rw http.ResponseWriter, resp *api.Response, cookies []*http.Cookie) {
```

Comment:
This method is not very useful. The caller can just call UpdateCookie in a loop directly.